lib/buffer.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable require-buffer */
 'use strict';
```

Comment:
I thought `'use strict'` had to go on the first line of either the file or function. Does that exclude comments?

/cc @domenic